remotes/fixuphelpers.go

```diff
@@ -91,7 +91,7 @@ func makeManifestWalker(ctx context.Context, sourceFetcher remotes.Fetcher,
 	scheduler := newErrgroupScheduler(ctx, cfg.maxConcurrentJobs, cfg.jobsBufferLength)
 	cleaner := func() {
 		cancel()
-		scheduler.drain()
+		_ = scheduler.drain()
```

Comment:
It's just my preference but I prefer to explicitly disable the linter for the line (with `//nolint:<RULE>`) rather than doing the `_ = ...` stuff. `nolint` is greppable, but the `_` is not.

Reply:
Ideally when we disable a rule for a line we would also add a simple comment saying why we disregard the rule.

Reply:
didn't know about //nolint, like it